pandas/tests/series/indexing/test_indexing.py

```diff
@@ -923,3 +923,14 @@ def test_getitem_2d_no_warning():
     series = pd.Series([1, 2, 3], index=[1, 2, 3])
     with tm.assert_produces_warning(None):
         series[:, None]
+
+
+def test_multiindex_at_get():
+    series = pd.Series([1, 2], index=[[1, 2], [3, 4]])
```

Comment:
can you also test .loc here (and below)

Reply:
I created new tests for these two, also in pandas/tests/indexing/test_scalar. I did not include the issue # as it only seems relevant to the .at tests. Let me know if any of this is off!

Reply:
I failed to notice that the rest of test_scalar combines tests for at and loc. I've combined the tests now to agree with the style of the rest of the file.